instrumentation/async-http-client/async-http-client-1-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/asynchttpclient/common/AsyncHandlerData.java

```diff
@@ -29,4 +29,11 @@ public static AsyncHandlerData create(
   public abstract Request getRequest();
 
   public abstract Instrumenter<Request, Response> getInstrumenter();
```

Comment:
The null-check below implies it can be null, if so we should probably annotate it with `@Nullable`.

```suggestion
  public abstract @Nullable Instrumenter<Request, Response> getInstrumenter();
```